src/trackers/USENET/search_helpers.py

```
@@ -0,0 +1,97 @@
+
+    return str(meta.basename_no_ext or title).strip()
+
+
+def parse_newznab_dupes(
+    response_text: str,
+    torrent_url: str | None = None,
+    *,
+    use_guid_attr_as_id: bool = False,
+) -> list[dict[str, Any]]:
+    dupes: list[dict[str, Any]] = []
+    response_xml = ElementTree.fromstring(response_text)
```

Ruff (0.15.21)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)